services/dcs.yaml

```diff
@@ -0,0 +1,28 @@
+name: dcs
+image: download-controller-service
+description: Interrogation Room Service
+summary: ""
+consumes:
+  events:
+    - topic: internal_file_registry
+      type: file_registered
+      config: files_to_register
```

Comment:
Is this to map this event to the config parameters of the service?

Reply:
Yes